dts/common/nordic/nrf54h20.dtsi

```diff
@@ -25,20 +25,23 @@
 			compatible = "arm,cortex-m33";
 			reg = <2>;
 			device_type = "cpu";
+			clocks = <&cpuapp_hsfll>;
 			clock-frequency = <DT_FREQ_M(320)>;
 		};
 
 		cpurad: cpu@3 {
 			compatible = "arm,cortex-m33";
 			reg = <3>;
 			device_type = "cpu";
+			clocks = <&cpurad_hsfll>;
 			clock-frequency = <DT_FREQ_M(256)>;
```

Comment:
we should at some point remove clock-frequency as it has become redundant

Reply:
outside the scope of this PR